### PR TITLE
Consistent naming for summary tables (#132)

### PR DESCRIPTION
**Pull request by rhiaro**
* [x] tender_summary_with_data (view) -> tender_summary (view)
* [x] tender_summary (table) -> tender_summary_no_data (table)
* [x] release_summary_with_data (view) -> release_summary (view)
* [x] release_summary (table) -> release_summary_no_data (table)
* [x] release_summary_with_checks (view) -> removed and checks column added to release_summary (view)
* [x] rename `data` column to `release` on release_summary (view)

### CODE REVIEW

**Comment by rhiaro** [comment]
[kingfisher-views: Follow consistent pattern for *_summary_no_data tables and *_summary views (and data column names) & Merge release_summary_with_checks table into (renamed) release_summary view.](https://trello.com/c/iSj7eGjc/236-kingfisher-views-follow-consistent-pattern-for-summarynodata-tables-and-summary-views-and-data-column-names-merge-releasesummary)

File: docs/database.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@ Foreign key relationships exist on all `tables <https://www.postgresql.org/docs/
 .. image:: _static/erd.png
    :target: _static/erd.png
 
-This diagram can help to identify JOIN conditions. For example, all tables can be joined with the ``release_summary``, ``release_summary_with_data`` and ``release_summary_with_data`` tables on the ``id`` column.
+This diagram can help to identify JOIN conditions. For example, all tables can be joined with the ``release_summary_no_data``, ``release_summary`` and ``release_summary_with_checks`` tables on the ``id`` column.
```
Comment (jpmckinney) [inline docs/database.rst:28]:
```suggestion
This diagram can help to identify JOIN conditions. For example, all tables can be joined with the ``release_summary_no_data`` and ``release_summary`` tables on the ``id`` column.
```

File: docs/database.rst [hunk 2]
```diff
@@ -106,22 +106,6 @@ release_summary
    :widths: 10, 10, 40
    :file: definitions/release_summary.csv
 
-release_summary_with_data
```
Comment (jpmckinney) [inline docs/database.rst:109]: We should add a section for `release_summary_no_data`

File: docs/database.rst [hunk 3]
```diff
@@ -207,14 +191,6 @@ tender_summary
    :widths: 10, 10, 40
    :file: definitions/tender_summary.csv
 
-tender_summary_with_data
```
Comment (jpmckinney) [inline docs/database.rst:210]: We should add a section for `tender_summary_no_data`

**Review by jpmckinney — COMMENTED** [review]
Reviewed docs only.

**Comment by odscjames** [comment]
I have done docs changes requested - and also updated the diagram.

But I have noticed a discrepancy - tender_summary_no_data has a foriegn key, and tender_summary_no_data does not. This explains why it does not appear on the new diagram.

**Comment by odscjames** [comment]
Did commit to add other foreign key.

**Comment by kindly** [comment]
This looks good to me.

I have a slight concern having a single `release_summary` table.  The reason for multiple tables was that the join to both checks and data tables are really large and I had performance concerns. However, this may not be a problem any more as the checks table has been shrunk.   
The other concern is that I think that `release_summary_with_data` is being used by analysts a lot and may cause them some changes to their scripts. However, as this simplifies future usage, for the sake of backwards compatibility, I think that it is still a good idea. 

So overall I am happy with it.

**Comment by rhiaro** [comment]
Thanks @kindly! Concerns about changing analysts templates and the size of the checks being added to release_summary were discussed in https://github.com/open-contracting/kingfisher-views/issues/132, so analysts are aware.

**Review by odscjames — APPROVED** [review]
As @kindly has put a comment saying he approves, I'll assume it's fine to press the approve button.

**Pull request merged**